_posts/custom-viz/boeing/counter.html

```diff
@@ -1,4 +1,4 @@
-<div class="counter">
-  <h4>Number of parts: <span class="part-total">0</span></h4>
-  <h4>Number of places: <span class="places-total">0</span></h4>
+<div class="boeing-counter">
```

Comment:
List items can only go inside either `<ul>` or `<li>` elements, so you'll need to adjust this.